Source/WebKit/Scripts/generate-serializers.py

```diff
@@ -31,6 +31,7 @@
 #
 # AdditionalEncoder - generate serializers for StreamConnectionEncoder in addition to IPC::Encoder.
 # CreateUsing - use a custom function to call instead of the constructor or create.
+# ConstructSubclass - use a subclass to construct the object. Do not include namespace.
```

Comment:
For any changes to the generator, add to Source/WebKit/Scripts/webkit/tests/TestSerializedType.serialization.in and run "make -C Source/WebKit/Scripts/webkit/tests"

Reply:
@achristensen07 it might be a good idea to document that in `generate-serializers.py`, it took me a while to figure out as well when I was making my recent change.

Reply:
I added a `watchlist` message, and modified `metadata/git_config_extension` to show it.